FirebaseAuth/Sources/Swift/MultiFactor/TOTP/TOTPSecret.swift

```diff
@@ -57,24 +61,34 @@ import Foundation
     @MainActor @objc(openInOTPAppWithQRCodeURL:)
     open func openInOTPApp(withQRCodeURL qrCodeURL: String) {
       if GULAppEnvironmentUtil.isAppExtension() {
-        // iOS App extensions should not call [UIApplication sharedApplication], even if
-        // UIApplication responds to it.
+        // App extensions should not call [UIApplication sharedApplication] or [NSWorkspace
+        // sharedWorkspace],
+        // even if they respond to it.
```

Comment:
nit: consolidate lines
```suggestion
        // sharedWorkspace], even if they respond to it.
```